src/internals/arc.js

```diff
@@ -608,10 +608,14 @@ extend(ChartInternal.prototype, {
 			}
 		}
 
-		main.selectAll(`.${CLASS.chartArc}`)
+		const gaugeTextValue = main.selectAll(`.${CLASS.chartArc}`)
 			.select("text")
 			.style("opacity", "0")
-			.attr("class", d => ($$.isGaugeType(d.data) ? CLASS.gaugeValue : ""))
+			.attr("class", d => ($$.isGaugeType(d.data) ? CLASS.gaugeValue : ""));
+
+		config.gauge_fullCircle && gaugeTextValue.attr("dy", `${Math.round($$.radius / 14)}`);
```

Comment:
@netil Could you explain meaning of `14`?

Reply:
It's the arbitrary number to center vertically.

Reply:
Ok I got it